### PR TITLE
makers - nim: Use full file path

### PR DESCRIPTION
**Pull request by JCavallo**
Removed trailing whitespaces as well.

### CODE REVIEW

File: autoload/neomake/makers/ft/nim.vim [hunk 1]
```diff
@@ -3,12 +3,13 @@ function! neomake#makers#ft#nim#EnabledMakers()
...
-    return {                          
-                \ 'exe': 'nim',                                      
-                \ 'args': ['--verbosity:0', '--colors:off', '-c', 'check'],
-                \ 'errorformat':                                     
-                \   '%I%f(%l\, %c) Hint: %m,' .                      
-                \   '%W%f(%l\, %c) Warning: %m,' .                   
-                \   '%E%f(%l\, %c) Error: %m'                        
-                \ }                                                  
+    return {
+                \ 'exe': 'nim',
```
Comment (blueyed) [inline autoload/neomake/makers/ft/nim.vim]: That is implied by the function name, but ok.

File: autoload/neomake/makers/ft/nim.vim [hunk 2]
```diff
@@ -3,12 +3,13 @@ function! neomake#makers#ft#nim#EnabledMakers()
...
+                \ 'args': ['--listFullPaths''--verbosity:0', '--colors:off',
```
Comment (blueyed) [inline autoload/neomake/makers/ft/nim.vim]: Missing command after first element.

File: autoload/neomake/makers/ft/nim.vim (hunk 1 above)
Comment (JCavallo) [inline autoload/neomake/makers/ft/nim.vim]: I just kept the previous version, but I prefer to have it explicitely anyway so...

**Comment by blueyed** [comment]
Thanks!

**Comment by JCavallo** [comment]
My pleasure !

Le jeu. 2 févr. 2017 20:17, Daniel Hahler <notifications@github.com> a
écrit :

**Pull request merged**